components/modal/index.css

```diff
@@ -21,6 +21,7 @@ governing permissions and limitations under the License.
   /* Distance between top and bottom of dialog and edge of window for fullscreen dialog */
   --spectrum-dialog-fullscreen-margin: 32px;
   --spectrum-dialog-max-height: 90vh;
+  --spectrum-dialog-max-width: 90vw;
```

Comment:
Okay so bonkers question but can we really be positive that the modal will always exist across the entire page? I mean, yes, it _should_ but there might be some inventive reason why it needs to exist in a subsection of the page. Is there a way to use container queries or percents so we are targeting a portion of it's container rather than a portion of the viewport?

Reply:
I would imagine the Popover dialog variant be a better choice in situations where the "dialog" is only existing in a portion of the page, but still worth thinking about it.

I did originally have 90% instead but the max-height is already `90vh` and this height is related to the functionality of the vertical scroll, so I thought `90vw` made sense here. 

Are we clear to use container queries? I thought that was still on hold.

Reply:
The other thing I was reading that is relevant here is that the `vw` take into account the side scroll bar while % would not, which is subtle and more so a design question as to if we want it to do that.

Reply:
in chatting with design it looks like this value will be replaced with a token when Modal is migrated and the token would need to use `%` over `vw` so changing to `%` for now.

Reply:
Good question - not sure about container queries yet - need to do some testing first.

Reply:
This is outdated but just to have a record of the decision: let's hold off on container queries at the moment and we can always come back and evaluate them after our migration & S2 initiatives